pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -91,8 +91,10 @@ def create_box(self, position, dimensions_list, name=None, matname=None):
         >>> box_object = hfss.modeler.primivites.create_box(origin, dimensions, name="mybox", matname="copper")
 
         """
-        assert len(position) == 3, "Position Argument must be a valid 3 Element List"
-        assert len(dimensions_list) == 3, "Dimension Argument must be a valid 3 Element List"
+        if len(position) != 3:
+            raise ValueError("Position Argument must be a valid 3 Element List")
```

Comment:
```suggestion
            raise ValueError("Position argument must be a valid three-element list.")
```
Error messages should be written as short but complete sentences following Google style.